Makefile.arduino

```diff
@@ -14,7 +15,7 @@
 			LWIP_LIB=liblwip$$((2+4*v6))-$$mss$$featname.a \
 			LWIP_LIB_RELEASE=../../lib/liblwip$$((2+4*v6))-$$mss$$featname.a \
 			LWIP_INCLUDES_RELEASE=../include \
-			TOOLS=../../../xtensa-lx106-elf/bin/xtensa-lx106-elf- \
+			TOOLS=$(PREFIX)/bin/xtensa-lx106-elf- \
```

Comment:
This will not work with the default relative path.
I checked with the arduino env the line below instead:
```
       TOOLS=$(abspath $(PREFIX)/bin/xtensa-lx106-elf-) \
```
please check if it works in your environment.

Reply:
mm that does not work for me
```
/bin/sh: /home/max/.platformio/packages/framework-arduinoespressif8266@src-31d658a59f41540201fc3726a1394910/tools/sdk/lwip2/builder/~/.platformio/packages/toolchain-xtensa/bin/xtensa-lx106-elf-gcc: No such file or directory
```
just using abspath does not handle `~` as it seems

Reply:
But it does work without `abspath` right ?